shared/src/main/scala/com/cibo/evilplot/plot/Heatmap.scala

```diff
@@ -101,4 +101,23 @@ object Heatmap {
     val colorBar = ScaledColorBar(colorStream.take(colorCount), minValue, maxValue)
     apply(data, colorBar)
   }
+
+  /** Create a heatmap using a continuous coloring.
+    * @param data The heatmap data.
+    * @param coloring The coloring to use.
+    */
+
+  def apply(data: Seq[Seq[Double]],
```

Comment:
I think `ScaledColorBar` does something different from `ContinuousColoring`, in that it picks n distinct colors rather than doing interpolation between them (I think we want to bring the interpolation behavior into the heatmap)

Reply:
The `HeatMapRenderer` needs a 'ScaledColorBar', so the options are either convert from continuous coloring (what I did), make a new custom HeatMapRenderer, or modify the existing HeatMapRenderer. Should I do one of those other two options? I'm also confused by what you mean by interpolation between them, I am using the `coloring` to choose the colors first before all the data gets mapped into the `ScaledColorBar`